CMakeLists.txt

```diff
@@ -28,7 +33,7 @@ include_directories(
 
 list(APPEND LIBS
     ${IBP_LIBRARY} ${GOP_LIBRARY} ${CZMQ_LIBRARY} ${TOOLBOX_LIBRARY}
-    ${JERASURE_LIBRARY} ${ZMQ_LIBRARY} ${FUSE_LIBRARIES}
+    ${JERASURE_LIBRARY} ${ZMQ_LIBRARY} ${FUSE_LIBRARIES} gssapi_krb5
```

Comment:
I think it may be a dependency of another dependency which would mean it is implicitly installed and thus dropping it "works". However if it is not a dependency-of-another-dependency across all platforms dropping it may break things in some situations but not others.